csv-validation-tests/src/main/bin/run-csvw-tests

```diff
@@ -0,0 +1,1128 @@
+echo "test001: Simple table"
```

Comment:
Is this taken from somewhere else? Do we need to correctly attribute the source of it and ensure that this file is licensed identically to where it came from?

Reply:
This is created automatically while running the GenerateTest script. Once #22 is merged in, ya both repositories will have the same lisence

Reply:
If it's generated, might be a good idea to leave it out of git and include it in the .gitignore.